framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -793,43 +809,42 @@ public Void visitTypeVariable(AnnotatedTypeMirror.AnnotatedTypeVariable type, Vo
-            Void r = scan(type.getLowerBound(), aVoid);
+            Void r = scan(type.getLowerBound(), func);
             type.getLowerBound().addAnnotations(primarys);
             visitedNodes.put(type, r);
 
             type.getUpperBound().removeAnnotations(primarys);
-            r = scanAndReduce(type.getUpperBound(), aVoid, r);
+            r = scanAndReduce(type.getUpperBound(), func, r);
             type.getUpperBound().addAnnotations(primarys);
             visitedNodes.put(type, r);
             return r;
         }
 
         @Override
-        protected Void scan(AnnotatedTypeMirror type, Void aVoid) {
+        protected Void scan(
+                AnnotatedTypeMirror type, Function<AnnotationMirror, AnnotationMirror> func) {
             for (AnnotationMirror anno :
                     AnnotationUtils.createAnnotationSet(type.getAnnotations())) {
-                AnnotationMirror newAnno =
-                        standardizeAnnotationIfDependentType(
-                                context, localVarPath, anno, removeErroneousExpressions);
+                AnnotationMirror newAnno = func.apply(anno);
                 if (newAnno != null) {
-                    // Standardized annotations are written into bytecode along with explicitly
-                    // written nonstandard annotations. (This is a bug.)
-                    // Remove the old annotation and add the new annotations. The new annotation for
-                    // both the nonstandard annotation and the standard annotation are equal with
-                    // respect to Object#equals, so only one new annotation will be added to the
-                    // type.
+                    // Computed annotations are written into bytecode along with explicitly written
+                    // annotations. (This is a bug.)
+                    // So remove the old annotation and add newAnno. newAnno for both the explicitly
+                    // written annotation and the original annotation are equal with respect to
+                    // Object#equals, so only one new annotation will be added to the type.
                     type.removeAnnotation(anno);
                     type.addAnnotation(newAnno);
                 }
             }
-            return super.scan(type, aVoid);
+            return super.scan(type, func);
         }
     }
 
     /**
      * Checks all Java expressions in the given annotated type to see if the expression string is an
      * error string as specified by {@link DependentTypesError#isExpressionError}. If the annotated
-     * type has any errors, an expression.unparsable.type.invalid error is issued at the errorTree.
+     * type has any errors, an expression.unparsable.type.invalid error is issued at {@code
+     * errorTree}.
      *
      * @param atm annotated type to check for expression errors
      * @param errorTree the tree at which to report any found errors
```

Comment:
The documentation of `errorElements` below says it returns elements, but actually it returns errors.